.gitmessage

```diff
@@ -0,0 +1,15 @@
+# [#NNN] (If applied, this commit will...) <subject> (Max 72 char)
```

Comment:
This template would need to be manually installed by the user or referred to via a `-t` option, right? We should somehow inform the users about the existence of this file.

Reply:
Maybe we could simply have a linter via a git commit hook or a GitHub action...

Reply:
I think having both. It is better to use the template when composing the original commit, that is when you need the prompt. Otherwise it needs amend, force push, etc.
Ideally we should configure it as a feature of the git repo, but I think it does need to be set up per repo, via `git config commit.template .gitmessage`